### PR TITLE
[Issue-13560] User avatar component

### LINKED ISSUE #13560: UI Component - User Avatar
This issue is to track the creation of the component: **User Avatar**

You can access the link at https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Mobile-Components?node-id=115%3A6787.

Please have a chat with the design team before starting with the component, and don't forget to add sizing to the issue

### PR DESCRIPTION
**Pull request by ibrkhalil**
fixes #13560 

status: ready

Progress so far
image

Update
image

### CODE REVIEW

File: src/quo2/components/user_avatar.cljs [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                            2
+                            0))]
+      [rn/view {:style {:background-color (if online?
+                                            colors/success-50
+                                            colors/neutral-40)
+                        :width dimensions
+                        :height dimensions
+                        :border-width border-width
+                        :border-radius dimensions
+                        :border-color "white"
```
Comment (J-Son89) [inline src/quo2/components/user_avatar.cljs]: should "white" be taken from colors/ file too? 🤔

File: src/quo2/screens/user_avatar.cljs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                 {:label "Online status"
+                  :key   :online?
+                  :type  :boolean}
+                 {:label "Status Indicator"
+                  :key   :status-indicator?
+                  :type  :boolean}
+                 {:label "Identicon Ring"
+                  :key   :ring?
+                  :type  :boolean}
+                 {:label "Full name seperated by space"
```
Comment (J-Son89) [inline src/quo2/screens/user_avatar.cljs]: nit: I think you have a typo here. Should be sep'a'rated

Comment (ibrkhalil) [reply src/quo2/screens/user_avatar.cljs]: You're totally right, Thanks!

File: src/quo2/components/user_avatar.cljs (hunk 1 above)
Comment (ibrkhalil) [reply src/quo2/components/user_avatar.cljs]: Done
Thanks

**Comment by siddarthkay** [comment]
Design issue 1 : The white border surrounding the online indicator needs to be thicker ;) 
Check screenshot below for comparison 

Screenshot 2022-08-22 at 6 31 07 PM

**Comment by siddarthkay** [comment]
Question/ Design Issue 2 : Should there be more options for sizes in this component considering we see 5 different sizes ( 16,20,24,28,32,48,80) in figma and only 3(Big,Small and Medium ) in the sandbox

Screenshot 2022-08-22 at 7 36 02 PM
?

**Comment by ibrkhalil** [comment]
> Question/ Design Issue 2 : Should there be more options for sizes in this component considering we see 5 different sizes ( 16,20,24,28,32,48,80) in figma and only 3(Big,Small and Medium ) in the sandbox
> 
> Screenshot 2022-08-22 at 7 36 02 PM ?

Done

**Comment by ibrkhalil** [comment]
> > Question/ Design Issue 2 : Should there be more options for sizes in this component considering we see 5 different sizes ( 16,20,24,28,32,48,80) in figma and only 3(Big,Small and Medium ) in the sandbox
> > Screenshot 2022-08-22 at 7 36 02 PM ?
> 
> Done

Done

**Pull request closed without merging**